appengine/analytics/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "express": "4.16.4",
```

Comment:
Would it be bad to try brining express up to `^` while you're in here?

Reply:
Only changing the deps that I recently updated, otherwise I'll have too many breaking changes.